docs/reference-guides/data/data-core-edit-site.md

```diff
@@ -182,6 +182,19 @@ _Returns_
 
 -   `boolean`: True if the list view panel should be open; false if closed.
 
+### isModalActive
```

Comment:
Do you think these "modals" handling (opening, closing, selectors) should be more part of the "interface" package/store? (and shared between edit post and edit store)?

Reply:
That's a great idea! I'll do that!